backend/tests/Squidex.Infrastructure.Tests/Squidex.Infrastructure.Tests.csproj

```diff
@@ -23,6 +23,7 @@
     <PackageReference Include="Google.Cloud.Storage.V1" Version="2.3.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
```

Comment:
Whats that?

Reply:
Reverted. There were issues while building "Squidex.Web.Tests", "Squidex.Infrastructure.Tests" and "Squidex.Domain.Users.Tests" projects, so I had to add these dependencies, but now it works without them as well. Probably I had to delete the NuGetFallbackFolder after installing 3.0 SDK, that might have caused this issue.